### PR TITLE
record likes on article endpoint 

### PR DESCRIPTION
**Pull request by beblicarl**
#### Pre-Submission Checklist

-   [x] Your pull request targets the `main` branch.
-   [x] Branch name is descriptive and follows the guidelines [here](https://github.com/andela/bestpractices/wiki/Git-naming-conventions-and-best-practices).
-   [x] All tests pass . Use `git commit --amend` to amend any fixes.

#### Checklist:

-   [x] All tests are passing.
-   [x] Tested changes locally.
-   [x] Improved documentation or readme where necessary.
-   [x] Your commit messages format follows our commit message guideline [here](https://github.com/andela/bestpractices/wiki/Git-naming-conventions-and-best-practices#commits).

#### What does this PR do?

record likes on article endpoint 
#### How should this be manually tested?

1. Start the server with `npm run dev`
2. Make a POST request with the sample data below to this endpoint `http://localhost:3000/api/v1/articles/13/likes`
```
{   
    "userId" : 4
}
```
3. This successfully returns the sample data below
```
{
    "status": "success",
    "data": {
        "message": "Article successfully liked",
        "userId": 4,
        "articleId": 13
    }
}
```

#### What are the relevant issues?
we had to record likes on article endpoint

### CODE REVIEW

**Comment by beblicarl** [comment]
@jocrah , kindly review this for me, thanks

File: src/routes/articles.js [hunk 1]
```diff
@@ -17,15 +18,18 @@ const {
 	 deletePostSchema,
 	 assignTagToArticleSchema,
 	 deleteArticleTagsSchema,
-	 queryArticleTagsSchema
+	 queryArticleTagsSchema,
+	 recordLikesOnPostSchema
```
Comment (jocrah) [inline src/routes/articles.js:22]: forgot to review this part, let's use `likePostSchema`

**Comment by jocrah** [comment]
kindly add the related documentation

File: src/routes/articles.js (hunk 1 above)
Comment (beblicarl) [reply src/routes/articles.js:22]: yes please

File: src/services/posts/like-post.test.js [hunk 2]
```diff
@@ -4,43 +4,92 @@ const likePost = require("./like-post")
...
 	})
-	it('should throw an error if Gif is already liked', async () => {
-		await fixtures.insertPostLike({
-			userId : user.id,
-			postId: post.id
+	describe('Gif', () => {
+		
+		let post
+		before(async ()=>{
+			user = await fixtures.insertUser()
```
Comment (jocrah) [inline src/services/posts/like-post.test.js:16]: not sure inserting a new user here affects the test?

Comment (beblicarl) [reply src/services/posts/like-post.test.js]: it was an oversight, i have rectified it now

**Review by jocrah — APPROVED** [review]

**Pull request merged**